modules/install-central-roxctl.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * installing/install_quick_roxctl.adoc
+:_module-type: PROCEDURE
+[id="install-central-roxctl_{context}"]
+= Running the Central installation scripts
+
+After you run the interactive instller, you can run the `setup.sh` script to install Central.
+
+.Procedure
+. Run the `setup` script to configure image registry access,
```

Comment:
* since you have it in backticks, should it be `setup.sh` ?
* End with a colon instead of a comma